tests/functional/u/unbalanced_tuple_unpacking.py

```diff
@@ -121,10 +137,21 @@ def my_sum(self):
 
     def sum_unpack_3_into_4(self):
         """Attempt to unpack 3 variables into 4"""
-        first, second, third, fourth = self # [unbalanced-tuple-unpacking]
+        first, second, third, fourth = self  # [unbalanced-tuple-unpacking]
         return first + second + third + fourth
 
     def sum_unpack_3_into_2(self):
         """Attempt to unpack 3 variables into 2"""
-        first, second = self # [unbalanced-tuple-unpacking]
+        first, second = self  # [unbalanced-tuple-unpacking]
         return first + second
+
+
+def my_function(mystring):
```

Comment:
The added test; The other changed content is from running `black` on the module.